src/lib/pluginUtils.ts

```diff
@@ -105,7 +105,7 @@ export function pluginName(plugin: IPluginState|IContractInfo, fallback?: string
       name = "Blockchain Interaction";
     }
   } else if (plugin.name === "ContributionRewardExt") {
-    name = rewarderContractName(plugin as IContributionRewardExtState);
```

Comment:
This is going to hide desirable aliases.  We should fix the alias in the migration repo